model/flow/timeout_certificate.go

```diff
@@ -0,0 +1,19 @@
+package flow
+
+// TimeoutCertificate represents a timeout certificate for some view.
+// A timeout certificate is a collection of special votes(called timeouts) which describe intent of committee to change active view.
+// Valid timeout certificates contain signatures from a super-majority of consensus committee members and
+// max(TOHighQCViews) == TOHighestQC.View
```

Comment:
We should be as precise as possible here
* I am worried that describing TimeoutObjects as votes might be confusing for some people who are not deeply familiar with the PaceMaker. 
* In our code base, the "timeouts" are called `TimeoutObject` and we should reference them with this name (to reduce ambiguity) 
* From my perspective, these validity conditions are incomplete
   ```golang
   // Valid timeout certificates contain signatures from a super-majority of consensus committee members and
   // max(TOHighQCViews) == TOHighestQC.View
   ```
   I would suggest to either _exhaustively_ list all validity conditions or not talk about validity conditions at all. 

```suggestion
// TimeoutCertificate proves that a supermajority of consensus participants want to abandon the specified View.
// At its core, a timeout certificate is an aggregation of TimeoutObjects, which individual nodes send to signal
// their intent to leave the active view.
```